src/test/resources/feature/SignUp.feature

```diff
@@ -0,0 +1,52 @@
+@Irina
+Feature: SignUp
+
+  Background:
+    Given "Home" is displayed
```

Comment:
Remove this step as you should handle it in Before Hook

Reply:
Done